crates/rspack_plugin_javascript/src/plugin/mod.rs

```diff
@@ -786,150 +795,203 @@ impl JsPlugin {
   ) -> Result<HashMap<String, Arc<dyn Source>>> {
     let mut inlined_modules_to_info: HashMap<String, InlinedModuleInfo> = HashMap::new();
```

Comment:
I was wondering if the key can be a direct `Identifier`. Its performance is supposed to be better than `string`.

Reply:
Done. Good suggestion.